vault/vault.go

```diff
@@ -40,30 +49,35 @@ type ClientCredential struct {
 type Configuration struct {
 	Credentials              ClientCredential
 	Tenant, TLD, URLTemplate string
+	Provider                 auth.Provider
 }
 
-// Vault provides access to secrets stored in Delinea DSV
+// Vault provides access to secrets stored in Thycotic DSV
```

Comment:
Probably this change should be reverted :)

Reply:
good catch!